src/main/com/fulcrologic/fulcro/components.cljc

```diff
@@ -868,7 +868,10 @@
                     (log/warn "String ref on " (component-name class) " should be a function."))
 
                   (when (or (nil? props) (not (gobj/containsKey props "fulcro$value")))
-                    (log/error "Props middleware seems to have the corrupted props for " (component-name class))))))
+                    (log/error "Props middleware seems to have the corrupted props for " (component-name class)))
+
+                  (when (sequential? (gobj/get props "fulcro$value"))
```

Comment:
Hm...`sequential?` will return false for a set...perhaps you mean `when-not (map? ...)`?

Reply:
fixed